ssl-config/src/main/java/com/palantir/remoting/ssl/KeyStoreConfiguration.java

```diff
@@ -49,12 +48,11 @@ public String type() {
 
     @Value.Check
     protected final void check() {
-        checkArgument(!path().equals(Paths.get("")), "path cannot be empty");
```

Comment:
ok, checking for empty string specifically. dunno when this will happen but sure, one fail-fast check